src/templates/RoomList/RoomList.tsx

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable sort-keys */
```

Comment:
do not use eslint-ignore ;)

Reply:
fixed in [08f6a45](https://github.com/dorm-social-room-manager/dsrm-frontend/pull/38/commits/08f6a4533a8460a854d4bdbe56059b6997dbc9ec)